packages/ozone/src/db/schema/index.ts

```diff
@@ -9,6 +9,7 @@ import * as signingKey from './signing_key'
 import * as communicationTemplate from './communication_template'
 import * as set from './ozone_set'
 import * as member from './member'
+import * as Setting from './setting'
```

Comment:
Just a small consistency thing.
```suggestion
import * as member from './member'
import * as setting from './setting'
```